packages/amplify_core/example/.metadata

```diff
@@ -0,0 +1,10 @@
+# This file tracks properties of this Flutter project.
```

Comment:
Do we have a concrete reason for including an example app for core?  iOS unit tests?

Reply:
As discussed offline, it is needed for iOS unit tests.